src/gui/MainWindow.cpp

```diff
@@ -399,6 +402,10 @@ void MainWindow::initGui()
     mUploadAction->setShortcut(Qt::SHIFT + Qt::Key_U);
     connect(mUploadAction, &QAction::triggered, this, &MainWindow::upload);
 
+    mCopyToClipboardAction->setText(tr("Copy to clipboard"));
```

Comment:
Here to the name should indicate that you copy it as Data Uri. Tooltips are not shown in file menu I think.